### ISSUE TITLE
Ghostfolio in Italiano

### ISSUE THREAD

**Issue by dtslvr** [ghostfolio/ghostfolio#3587]
As Ghostfolio continues to evolve, some translations are still missing. We need your help to make Ghostfolio fully accessible to Italian-speaking users!

Here’s how you can participate in the translation work:

1. Open the translation file at https://github.com/ghostfolio/ghostfolio/blob/main/apps/client/src/locales/messages.it.xlf
2. Search for entries with `state="new"`
3. Translate the content of the `<target>` element
4. Change the state from `"new"` to `"translated"`
5. Create a pull request with your translations

You don’t need to translate everything at once. Even small contributions are incredibly valuable and easier for us to review.

Your efforts will make a big difference. Many thanks in advance for your help!

**Comment by sghebuz** [comment]
Hello, I can work on the Italian version, considering I have never worked on the translation of a project I have just a quick question: I see that there are some entries with `state="new"` that are already translated, shall I just verify them and then change their state to `"translated"`? thanks

**Comment by dtslvr** [comment]
> I see that there are some entries with `state="new"` that are already translated, shall I just verify them and then change their state to `"translated"`?

Exactly. I don't know why the `state` sometimes changes while the translation remains the same.

**Comment by sghebuz** [comment]
I'm sorry, is there a recommended tool to edit the xlf file? I tried with poedit but it's rewriting the xml too much. thanks

**Comment by dtslvr** [comment]
> I'm sorry, is there a recommended tool to edit the xlf file?

You could either use a conventional text editor or edit the file directly on [GitHub](https://github.dev/ghostfolio/ghostfolio/blob/main/apps/client/src/locales/messages.it.xlf).

**Comment by geooner** [comment]
@dtslvr Hi, I have made the translations in #4681 . Hope it was helpful.

**Comment by fabiocarlos97** [comment]
@dtslvr  Hello, I have implemented the recent requested Italian translation changes. Please review PR #4717. Thank you

**Comment by dtslvr** [comment]
I have extended the instructions by a changelog entry.

**Comment by MImran2002** [comment]
@dtslvr Hi I am from codepath A301 class can I work on this issue? Is this available for me to take?

**Comment by dtslvr** [comment]
> can I work on this issue?

Sure @MImran2002, please create a pull request with your translations. As this is an ongoing issue, we don't assign it to you now.
